### ISSUE TITLE
NotImplementedError

### ISSUE THREAD

**Issue by yuanzhi-liang** [c0nn3r/RetinaNet#13]
did anyone meet this problem?
there are some answer in PyTorch Forums said that the code should have a forward part, so how to add it in this code?

**Comment by akira-l** [comment]
I run the train_coco.py directly. Maybe I need other operation ?

**Comment by akira-l** [comment]
addition:
Instead of COCO dataset, I want to use retinanet in wider face. So I change the data input like this:
transform = transforms.Compose([
    transforms.ToTensor(),
    transforms.Normalize((0.485,0.456,0.406), (0.229,0.224,0.225))
])

trainset = ListDataset(root="./../../wider face/WIDER_train/images",
                       list_file="./../../wider face/wider_face_split/wider_face_train_bbx_gt.txt", train=True, transform=transform, input_size=600)
train_loader = torch.utils.data.DataLoader(trainset, batch_size=1, shuffle=True, num_workers=8, collate_fn=trainset.collate_fn)
